linux-4.19.defconfig

```diff
@@ -86,7 +91,7 @@ CONFIG_CMA_SIZE_MBYTES=5
 CONFIG_CONNECTOR=m
 CONFIG_OF_CONFIGFS=y
 CONFIG_BLK_DEV_LOOP=y
-CONFIG_BLK_DEV_RAM=y
+CONFIG_BLK_DEV_RAM=m
```

Comment:
This is an interesting change. If `m` works, and I believe that it does, then I'd expect `n` to work as well. I.e., nothing uses a ramfs. If I could go back in time, I would have turned ramfs off, since tmpfs's are better and all official Nerves RAM-based filesystems use them. 

Could you remove this change in this PR? At the moment, I think that I'm for a separate PR that turns off ramfs support completely, and that would be good to call out in case one of our users really likes them. I'd also be curious about why.

Reply:
Done